db/migrate/201509151000_create_qualifying_comments_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQualifyingCommentsTable < ActiveRecord::Migration
+  def change
+  	create_table :qualifyings do |t|
```

Comment:
I had something like that initially, but it told me "qualifyings" couldn't be found. I assumed it took the singular I was using (qualifying) and was looking for a table named qualifying + s. Can you tell me what's going on?